### DISCUSSION TITLE
Kitsu statistics collecting

### DISCUSSION THREAD

**Discussion by frankrousseau** [cgwire/kitsu#1233] [Ideas]
For the community, it will be to know better who is using Kitsu as an open-source installation: is it a small group made of the active members of this channel, or is it something much bigger like these numbers show (probably something in between)?

That's why we will spend some time to add a statistic tracker. You will be able to disable it with a flag, and the results will be shared publicly. We will collect the following information:

- organisation ID in Kitsu (to avoid duplicates)
- number of video previews
- number of picture previews
- number of model previews
- number of comments
- date of the last comment

What do you think about it?

**Comment by NicoPennec** [comment]
I frequently come across the term "telemetry" used to refer to this feature.
For example:
- https://code.visualstudio.com/docs/getstarted/telemetry
- https://nextjs.org/telemetry
- https://developer.hashicorp.com/terraform/cdktf/telemetry

**Comment by PeteDraper** [comment]
A lot of us work with the system not exposed to the internet due to security so this may be an issue. I would suggest also giving an option to export this data as a csv to hand over if this is the case. Concern for any studio is what data is being shared and if that data is identified to a specific company - anonymised data would be best to submit, but you’ll be hard pressed getting any large studio to share this information.

**Reply by frankrousseau** [reply]
We will anonymize the data as much as possible.
The data won't be collected for our customers, so large studios are fine.

**Comment by frankrousseau** [comment]
We added the feature. Telemetry data are collected on database upgrade. Everyting stays anonymous. The code can be audited here: https://github.com/cgwire/zou/pull/762

**Comment by frankrousseau** [comment]
I close the discussion. Feel free to reopen it if needed.
